scripts/irods/configuration.py

```diff
@@ -248,7 +248,7 @@ def validate_configuration(self):
                 'dict': self.server_config,
                 'path': paths.server_config_path()
             },
-            'VERSION': {
+            'VERSION': { # TODO Investigate
```

Comment:
This should probably be changed because it is part of the dict used to validate the configuration files on startup.

Thoughts?

Reply:
Changed it.